### DISCUSSION TITLE
Docker WAIT_START

### DISCUSSION THREAD

**Discussion by FLUXtRATEX** [Security-Onion-Solutions/securityonion#3143] [Unsupported Versions]
Hello All, 

I have recently installed Security Onion on a virtual box instance. I am setting this up for a lab environment to learn the system, and am not a Linux guru. I continue to get Docker WAIT_START when I run sudo so-status. Can someone advise?

Thanks, 

![image](https://user-images.githubusercontent.com/79545183/109223270-5f1e2200-7788-11eb-8794-4dadf2126ed8.png)

**Comment by weslambert** [comment]
What version of Security Onion are you using, and what kind of specs are being used for the VM and the host?

**Comment by Gleydstone** [comment]
I have the same problem.
I am using Security Onion version 2.3.21 on Vmware ESXI 6.7
 Docker -------------------------------------------------------------------------------- [ ERROR ] 
It happened after I run the command soup and restart the VM.
[soup.log](https://github.com/Security-Onion-Solutions/securityonion/files/6393729/soup.log)
